package.json

```
@@ -48,7 +49,9 @@
     "test:unit:update": "yarn run test:unit --updateSnapshot",
     "test:a11y": "LC_ALL=en_US.UTF-8 jest --config .jest/a11y.config.ts",
     "prepare": "husky install",
-    "size": "yarn run build && size-limit"
+    "size": "yarn run build && size-limit",
+    "visual:update": "yarn run build:storybook-visual && yarn loki update",
+    "visual:test": "yarn loki test"
```

```suggestion
    "test:visual:update": "yarn run build:storybook-visual && yarn loki update",
    "test:visual": "yarn loki test"
```

To stay consistent with other test commands